src/NuGet.Core/NuGet.Commands/RestoreCommand/RestoreRequest.cs

```diff
@@ -22,11 +23,21 @@ public class RestoreRequest
 
         private Lazy<LockFile> _lockFileLazy;
 
+        public RestoreRequest(
```

Comment:
Are there any tests for this API?

Maybe we should add some basic ones?

Reply:
There are 8 tests for the old constructor in https://github.com/NuGet/NuGet.Client/blob/dev/test/NuGet.Core.Tests/NuGet.Commands.Test/RestoreCommandTests.cs

![image](https://user-images.githubusercontent.com/52756182/122306665-bb36be80-cebd-11eb-8114-fc9646419561.png)